docs/RELEASE.md

```diff
@@ -0,0 +1,154 @@
+# Release
+
+The outputs of a release are the `polkadot` and `polkadot-parachain` node binaries, the runtimes for Westend & Rococo and their system parachains, and new crate versions published to `crates.io`.
+
+# Setup
+
+We have two branches: `master` and `stable`. `master` is the main development branch where normal merge requests are opened. Developers need to mostly only care about this branch.  
+The `stable` branch contains a version of the code that is ready to be released. Its contents are always audited. Merging to it is restricted to [Backports](#backports).
```

Comment:
https://github.com/paritytech/polkadot-sdk/pull/2490#discussion_r1454579198 does this answer your q?

Reply:
Not really what would the workflow for relay chain devs look like? Like which branch would we usually base our PRs on? If I read this document right, it would be master and then we would backport to stable? And this for all Polkadot PRs?

I feel like the monorepo is biting us here a bit. From the perspective of Polkadot a more consuming perspective would make sense in my opinion. Basically we just use substrate as a normal dep, with its releases.  Develop Polkadot and whenever we actually need a new substrate feature (or need security fixes) we update our dependency.

Reply:
>I feel like the monorepo is biting us here a bit. From the perspective of Polkadot a more consuming perspective would make sense in my opinion

Could be... but i think if we go with audited nightly then it should be very close to what we currently have.  
But its not really that nice, since currently we update the Runtimes to a proper Polkadot-SDK release, in the future we would update then to a nightly release.